StarkInfra/StarkInfra/CreditNote/CreditNote.cs

```diff
@@ -112,37 +113,38 @@ public partial class CreditNote : Resource
         /// <list>
         ///     <item>rebateAmount [integer, default null]: credit analysis fee deducted from lent amount. ex: rebateAmount=11234 (= R$ 112.34)</item>
         ///     <item>tags [list of strings, default null]: list of strings for reference when searching for CreditNotes. ex: tags= new List<string>{ "employees", "monthly" }</item>
+        ///     <item>expiration [integer]: time interval in seconds between due date and expiration date. ex: 123456789</item>
```

Comment:
faltou o valor default do **expiration** e a docstring dele deveria ser: `time interval in seconds between scheduled date and expiration`